src/content/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables.mdx

```diff
@@ -64,13 +64,11 @@ CORECLR_PROFILER_PATH="C:\Program Files\New Relic\.NET Agent\netcore\NewRelic.Pr
 
 Note: There is a subtle difference in the behavior of the .NET Framework agent vs. the .NET Core agent regarding having profiling enabled. For the .NET Framework agent, even when profiling is enabled in the environment of a .NET Framework process, it will only be instrumented by the agent if one of the following conditions are met:
 
-* The application is hosted in IIS and
-  * application pools are configured to be instrumented by default, or
-  * the application pool hosting the application is configured to be instrumented explicitly
+* The application is hosted in IIS (a) application pools are configured to be instrumented by default, or (b) the application pool hosting the application is configured to be instrumented explicitly.
```

Comment:
I'm having difficulty parsing this as a whole, and specifically what is required and what are the choices